CONTRIBUTING.md

```diff
@@ -14,7 +14,6 @@
     - [Environment Preparations](#environment-preparations)
     - [Project Layout](#project-layout)
     - [NPM Scripts](#npm-scripts)
-    - [Coding Guidelines](#coding-guidelines)
```

Comment:
No more coding guidelines?

Reply:
It's previous content is not relevant anymore.